### ISSUE TITLE
TableExporter.getSqlCreateStrings fails with NPE

### ISSUE THREAD

**Issue by meltsufin** [GoogleCloudPlatform/google-cloud-spanner-hibernate#27]
Many of the documentation tests seem to be failing with an NPE in `TableExporter.getSqlCreateStrings`.

```
Caused by: java.lang.NullPointerException
	at com.google.cloud.spanner.hibernate.SpannerTableExporter.getSqlCreateStrings(SpannerTableExporter.java:56)
	at com.google.cloud.spanner.hibernate.SpannerTableExporter.getSqlCreateStrings(SpannerTableExporter.java:35)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.createFromMetadata(SchemaCreatorImpl.java:316)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.performCreation(SchemaCreatorImpl.java:166)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.doCreation(SchemaCreatorImpl.java:135)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.doCreation(SchemaCreatorImpl.java:121)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:155)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:72)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:309)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:462)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:938)
	... 13 more
```

**Comment by ChengyuanZhao** [comment]
@meltsufin which set of tests does this one fall under (the --tests <test_name>)? I think we would be throwing an UnsupportedOperationException here instead, but want to be sure..

**Comment by meltsufin** [comment]
Sure, try this one:
```
python3 run-tests.py SQLTest.test_sql_jpa_all_columns_scalar_query_example
```

**Comment by ChengyuanZhao** [comment]
closed by #28
